core/src/main/resources/lib/form/entry.jelly

```diff
@@ -89,14 +94,9 @@ THE SOFTWARE.
         <j:if test="${attrs.help!=null}">
           <f:helpArea />
         </j:if>
-        <j:if test="${!empty(attrs.description)}">
```

Comment:
reverting the changes in this file is enough to make `hudson.RelativePathTest#testRelativePath` pass and probably most of the rest of the test failures